vpro-shared-test/pom.xml

```diff
@@ -45,6 +50,10 @@
       <artifactId>junit-jupiter-params</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
+    </dependency>
```

Comment:
It seems illogical to add dependencies to vpro-shared-test if those are not used in the module itself.

As far as i can see the new module does not use anything from vpro-shared-test either, just using it's transitive dependencies.

Reply:
See my commit on this branch.

Reply:
Well I disagree, but will move it. Shared test can prevent down stream modules from the need to manage their own test dependencies. Hamcrest all is very general generic test resource. You probably want to use this on many locations.